frontend/src/pages/VendorDashboard.jsx

```
@@ -0,0 +1,371 @@
+  const handleDeleteProperty = async (id) => {
+    if (window.confirm("Are you sure you want to delete this property?")) {
+      try {
+        const response = await axios.delete(`http://localhost:4000/api/vendor/deleteProperty/${id}`, 
+          { withCredentials: true }
+        );
+        
+        if (response.data.success) {
+          setProperties(prev => prev.filter(property => property._id !== id));
+          alert("Property deleted successfully!");
+        } else {
+          alert("Failed to delete property: " + response.data.error);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)